LibreHardwareMonitor/UI/PlotPanel.cs

```diff
@@ -151,7 +153,7 @@ private PlotModel CreatePlotModel()
             _timeAxis.Minimum = 0;
             _timeAxis.AbsoluteMaximum = 24 * 60 * 60;
             _timeAxis.Zoom(
-              _settings.GetValue("plotPanel.MinTimeSpan", 0.0f),
+              _settings.GetValue("plotPanel.MinTimeSpan", double.NaN),
```

Comment:
This change breaks rendering graphs if never plotted a graph before. I'll do a cleanup and merge, thanks.